### PR TITLE
Case Studies

### PR DESCRIPTION
**Pull request by itsMapleLeaf**

### CODE REVIEW

File: src/content/caseStudies/contentful-share-pear/index.mdx
Comment (itsMapleLeaf) [inline src/content/caseStudies/contentful-share-pear/index.mdx:18]: Not sure what the best way to accomplish this styling is, so I just hardcoded it in the content for now 😅

**Review by itsMapleLeaf — COMMENTED** [review]
I also think I should wait until the assets PR is merged before continuing on this

File: src/pages/case-studies/[slug].astro
Comment (tony-sull) [inline src/pages/case-studies/[slug].astro:54]: make sure to pull these comments out before merging, Astro will keep all of them in the DOM 😅

File: src/components/Blockquote.astro
Comment (tony-sull) [inline src/components/Blockquote.astro:5]: Do we use this component anywhere else yet?

I usually try to keep spacing between elements in the container itself, but if we always want a top margin here it might be a great case for `margin-trim` to ignore it if this is the first child

File: src/pages/case-studies/[slug].astro
Comment (tony-sull) [inline src/pages/case-studies/[slug].astro:1]: Are there many differences between this design and the blog post layout?  If they're pretty close it might be worth sharing a `ProseLayout.astro` or similar with most of the logic for cover images, post headers, etc

Comment (itsMapleLeaf) [reply src/pages/case-studies/[slug].astro:1]: I had the same thought, but I didn't want to abstract eagerly

File: src/components/Blockquote.astro
Comment (itsMapleLeaf) [reply src/components/Blockquote.astro:5]: Yeah that's a good point, I'll figure out some other way to accomplish this

**Comment by ximenavf92** [comment]
- The gradient colors for the hero are: `#6CE486` to `#4783B5`
- I also added a black layer to it at 20% opacity to improve contrast.

image
image

File: src/pages/case-studies/[slug].astro
Comment (delucis) [inline src/pages/case-studies/[slug].astro:158]: Looks like this could do with some love on mobile:
image

Off the top of my head: flip Houston down to the bottom and add some `py`?

**Comment by ximenavf92** [comment]
## CTA at the bottom
image image

- This needs to be the same width as the content above it.
- Let's add some top margin to separate from blog content a bit. Looks like `3rem` may do it. 
- Houston is a bit blurry. Idk if we usually use SVGs for this. Dropping on Discord an SVG, a 3x version of the image, and a full uncropped version for mobile unless we can move him to the button as we do on Astro.new; I know it's set up differently so whatever is easier.

File: src/pages/case-studies/[slug].astro
Comment (ximenavf92) [reply src/pages/case-studies/[slug].astro:158]: Yes! Agree with Chris. If we can move him to the bottom, that would be great if not I sent you on Discord an uncropped version of Houston that we can use in this layout you have. 

Comments on this current layout for mobile:
- Tighten up the spacing between elements (Houston, heading, text, button)
- Add top and bottom padding to give some breathing room

Let's make it look more like this:
image

If we can move Houston flush to the bottom, that would be lovely, like in Astro.new. If not, swap out for the uncropped image. 

[houston_wink-cta.zip](https://github.com/withastro/astro.build/files/11826652/houston_wink-cta.zip)

**Comment by ximenavf92** [comment]
## Hero header on mobile

- Let's make that logo smaller so it has some breathing room on the sides. 

### **Current**
image

### **Goal**
image

Also… I was playing with the gradient, and I think this should do it for this one. 

```css
    background-image: linear-gradient(120deg,#6CE486 0%,#4783B5 100%);
    filter: opacity(0.85);
```

**Comment by ximenavf92** [comment]
## Pull quote on mobile


Let's reduce the font size on mobile for this, **make it `20px`** so that it looks more like the screenshot below and does not take up that much space which will also increase the line length for readability.

image

**Comment by ximenavf92** [comment]
## Blog index card updates


**_I feel like this could be a part of this PR:_**

We are gonna add badges to the Case Study posts only in that feed. As such: 
image

Notice how the title and description are now stacked vertically and take up the full width of the card. I talked to Mark, and we want to make this the standard for all the cards in our blog. This should fix the current wonkiness with articles that have longer titles and/or descriptions.

File: src/pages/case-studies/[slug].astro
Comment (itsMapleLeaf) [reply src/pages/case-studies/[slug].astro:158]: Strange, Houston was _supposed_ to be on the bottom here, but I guess that broke somehow 🫠

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (clearlyTHUYDOAN) [inline src/content/caseStudies/contenda-share-pear/index.mdx:29]:
```suggestion
When Contenda needed to share draft blog posts publicly with other people, they turned to Astro and Netlify as their solution. This combination of technologies let them stay laser focused on their goal of making collaboration and content creation easy.
```

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (clearlyTHUYDOAN) [inline src/content/caseStudies/contenda-share-pear/index.mdx:41]:
```suggestion
Cassidy is Contenda’s CTO. Having previously worked at Netlify plus being an active developer in the Astro community, she saw an opportunity to use Astro with Netlify’s on-demand builders. In addition to meeting their requirements, the main benefit of choosing Astro was its flexibility. With Astro, popular JavaScript frameworks like React are available, but optional. A site can be built using just HTML, CSS and, if necessary, vanilla JavaScript. This enables team members with less web development experience, such as product designers and content authors, to edit and maintain the site often without any developer intervention required.
```

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (clearlyTHUYDOAN) [inline src/content/caseStudies/contenda-share-pear/index.mdx:47]:
```suggestion
Contenda found success in Astro’s Discord by enlisting the help of the core team and the community. From there, building was fast because the developer experience was good.
```

**Review by clearlyTHUYDOAN — DISMISSED** [review]
Copy edit changes.

File: src/components/Blockquote.astro
Comment (itsMapleLeaf) [reply src/components/Blockquote.astro:5]: I looked, and this PR is the first usage of this component 😅 I guess I'll just keep it for now?

**Comment by itsMapleLeaf** [comment]
> ## Blog index card updates
> **_I feel like this could be a part of this PR:_**
> 
> We are gonna add badges to the Case Study posts only in that feed. As such: image
> 
> Notice how the title and description are now stacked vertically and take up the full width of the card. I talked to Mark, and we want to make this the standard for all the cards in our blog. This should fix the current wonkiness with articles that have longer titles and/or descriptions.

I'll do this in a separate PR

File: src/pages/case-studies/[slug].astro
Comment (itsMapleLeaf) [reply src/pages/case-studies/[slug].astro:1]: The layout ended up changing in _lots_ of ways, so I'll keep it duplicated 😇

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (clearlyTHUYDOAN) [inline src/content/caseStudies/contenda-share-pear/index.mdx:3]:
```suggestion
description: "Building Public-Sharing of Blog Drafts in One Month"
```

Comment (delucis) [reply src/content/caseStudies/contenda-share-pear/index.mdx:3]: This still needs fixing to read “One” right?

Comment (clearlyTHUYDOAN) [reply src/content/caseStudies/contenda-share-pear/index.mdx:3]: Yup, bumping so Darius sees and commits! I thought it got lost in git history or something.

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (clearlyTHUYDOAN) [inline src/content/caseStudies/contenda-share-pear/index.mdx:3]:
```suggestion
description: "How Astro Helped Contenda Build a Content-Sharing Platform for Everyone"
```

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (clearlyTHUYDOAN) [inline src/content/caseStudies/contenda-share-pear/index.mdx:6]:
```suggestion
  description: "How an AI content company used Astro and Netlify's on-demand builders to enable users to share their generated blog posts with each other - all in one month."
```

**Review by clearlyTHUYDOAN — COMMENTED** [review]
Just dropped a couple new copy updates and bumped an old copy change that got lost in the mix.

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (clearlyTHUYDOAN) [inline src/content/caseStudies/contenda-share-pear/index.mdx:3]:
```suggestion
description: "How Astro Helped Contenda Build a Content-Sharing Platform in One Month"
```

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (clearlyTHUYDOAN) [inline src/content/caseStudies/contenda-share-pear/index.mdx:5]:
```suggestion
  title: "Content for Everyone: How Astro Helped Contenda Build a Content-Sharing Platform in One Month"
```

**Review by clearlyTHUYDOAN — COMMENTED** [review]
Another copy update! Replacing the redundant "for everyone" between title and description.

**Comment by ximenavf92** [comment]
## Latest OG image

og-image-cs-contenda

**Comment by FredKSchott** [comment]
Review Comments / Action Items:
- @clearlyTHUYDOAN decide on final URL (`/contenda-share-pear/`)?
- @itsMapleLeaf add case studies to show up in the blog index list, mixed in with blog posts (https://deploy-preview-718--astro-www-2.netlify.app/blog/)
- @clearlyTHUYDOAN check if original screenshot had low quality / pixelated look, either re-attempt to screenshot in higher quality or ask Darius to fix the image optimization

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (FredKSchott) [inline src/content/caseStudies/contenda-share-pear/index.mdx:52]:
```suggestion
In addition to meeting their requirements, the main benefit of choosing Astro was its flexibility. With Astro, popular JavaScript frameworks like React are available, but optional. A site can be built using just HTML, CSS and, if necessary, vanilla JavaScript. This enables team members with less web development experience, such as product designers and content authors, to edit and maintain the site often without any developer intervention required.
```

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (FredKSchott) [inline src/content/caseStudies/contenda-share-pear/index.mdx:54]:
```suggestion
Cassidy (Contenda CTO) saw an opportunity to pair Astro with Netlify’s on-demand builders. Astro was the obvious choice because it had a component that both natively supported Netlify’s builders and rendering markdown to HTML. It also helped that Contenda had previously chosen Astro for their main marketing website so the technology was familiar enough for the rest of the team to jump in.
```

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (FredKSchott) [inline src/content/caseStudies/contenda-share-pear/index.mdx:58]:
```suggestion
Contenda found success in Astro’s developer community. The team was able to enlist the help of the core team and active contributors in [Astro's community Discord server](https://astro.build/chat). From there, building Share Pear moved quickly.
```

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (FredKSchott) [inline src/content/caseStudies/contenda-share-pear/index.mdx:60]: duplicate paragraph of above?

**Review by FredKSchott — APPROVED** [review]
Looks good, left some small improvements. Most comments are minor except for one duplicate paragraph towards the end of the post

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (clearlyTHUYDOAN) [reply src/content/caseStudies/contenda-share-pear/index.mdx:32]: lmao "dramatic". yeah it's to plug them. could be removed! I think we already have another Grady quote as well (that one's about Astro though, which is muy bien).

Comment (clearlyTHUYDOAN) [inline src/content/caseStudies/contenda-share-pear/index.mdx:32]:
```suggestion
```

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (clearlyTHUYDOAN) [reply src/content/caseStudies/contenda-share-pear/index.mdx:58]: Approved ✅

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (clearlyTHUYDOAN) [reply src/content/caseStudies/contenda-share-pear/index.mdx:54]: Approved ✅

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (clearlyTHUYDOAN) [reply src/content/caseStudies/contenda-share-pear/index.mdx:52]: Approved ✅

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (clearlyTHUYDOAN) [reply src/content/caseStudies/contenda-share-pear/index.mdx:50]: Approved ✅

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (clearlyTHUYDOAN) [inline src/content/caseStudies/contenda-share-pear/index.mdx:7]:
```suggestion
publishDate: "June 29, 2023"
```

**Comment by itsMapleLeaf** [comment]
Drafting to include the post in the post list

File: src/content/caseStudies/contenda-share-pear/index.mdx
Comment (clearlyTHUYDOAN) [inline src/content/caseStudies/contenda-share-pear/index.mdx:72]:
```suggestion
[Share Pear](https://github.com/Contenda-Team/share-pear) has been online for about a month and is used by both the open-source community and customers of Contenda that [use their app](https://signup.contenda.co/). At time of writing, 30% of their users share blogs using the Share Pear method and this feature has become one of the top 3 most used on the platform.
```

**Pull request merged**
